### DISCUSSION TITLE
Run onedrive as different user

### DISCUSSION THREAD

**Discussion by kpatzwald** [abraunegg/onedrive#1910] [Q&A] [answered]
Hi,

is it possible to change the ownership of the downloaded files in the OneDrive-Directory? I want to use the OneDrive-Directory as external storage for my nextcloud-Server. Therefore it needs the group www-data.

Or is it possibe to run OneDrive as user www-data?

Best regards
Kay

**Comment by abraunegg** [comment, marked as answer]
@kpatzwald 

> is it possible to change the ownership of the downloaded files in the OneDrive-Directory? I want to use the OneDrive-Directory as external storage for my nextcloud-Server. Therefore it needs the group www-data.
> 
> Or is it possibe to run OneDrive as user www-data?

Your only option is to run a systemd service on your server as that user.

You will need to investigate how to do this, however:
* Login to your server
* su to root
* Manually create a systemd service file
  *  Edit the systemd file to include `User=<username>` and `Group=<group>` that you want the service to run as

Google Search: https://www.google.com/search?q=configure+a+systemd+service+as+specific+user
